rotkehlchen/chain/scroll/modules/scroll_airdrop/decoder.py

```diff
@@ -49,10 +54,42 @@ def _decode_airdop_claim(self, context: 'DecoderContext') -> 'DecodingOutput':
 
         return DEFAULT_DECODING_OUTPUT
 
+    def _decode_scroll_batch_airdrop(
+            self,
+            context: 'DecoderContext',
+    ) -> 'DecodingOutput':
+        """Decodes SCR token airdrop claims from Scroll's offchain distributor.
+
+        Handles batch distributions for GitHub/email-based claims that are processed
+        periodically rather than instant on-chain claims.
+
+        See https://scroll-faqs.gitbook.io/faq#23-when-will-i-receive-my-scr-tokens-after-claiming
+        """
+
+        if context.tx_log.topics[0] != EXECUTION_SUCCESS_TOPIC:
+            return DEFAULT_DECODING_OUTPUT
+
+        for event in context.decoded_events:
+            if (
+                    event.address == SCROLL_OFFCHAIN_TOKEN_DISTRIBUTOR and
+                    event.event_subtype == HistoryEventSubType.NONE and
+                    event.event_type == HistoryEventType.RECEIVE and
+                    event.asset == A_SCR
+            ):
+                event.counterparty = CPT_SCROLL
+                event.event_subtype = HistoryEventSubType.AIRDROP
+                event.extra_data = {AIRDROP_IDENTIFIER_KEY: 'scroll'}
+                event.notes = f'Claim {event.balance.amount} SCR from scroll airdrop'
```

Comment:
This is not really a claim but more like `Receive {event.balance.amount} SCR from scroll airdrop`